src/main/features/dashboard/views/macro/caseHistory.njk

```diff
@@ -1,44 +1,358 @@
+    'settled, or they can proceed with it.') }}
+  </p>
+{% endmacro %}
+
+{% macro _claimRejectedFreeMediationNOForDefendant(claim) %}
+  <h2 class="heading-medium">{{ t('Your response to the claim') }}</h2>
+
+  <p>
+    {{ t('You have rejected the claim. We’ll send you a directions questionnaire – this is a form which you must '
+    + 'complete to tell us more about the claim.') }}
+  </p>
+
+  <p>
+    {{ downloadResponse(claim, DashboardUrlHelper.getStatusUrl(claim), 'Download your Response') }}
+  </p>
+{% endmacro %}
+
+{% macro _claimRejectedFreeMediationYESForDefendant(claim) %}
+  <h2 class="heading-medium">{{ t('Your response to the claim') }}</h2>
+  <p>
+    {{ t('You have rejected the claim. You’ve suggested mediation.') }}
+  </p>
+
+  <p>
+    {{ t('We’ll ask {{ claimantName }} if they agree to take part in mediation.',
+    {claimantName: claim.claimData.claimant.name}) }}
+  </p>
+
+  <p>
+    {{ t('If they say yes, we’ll arrange a date for mediation. '
+    + 'If they say no we’ll send you a directions questionnaire - '
+    + 'this is a form you complete to tell us more about the claim.') }}
+  </p>
+
+  <p>
+    {{ downloadResponse(claim, DashboardUrlHelper.getStatusUrl(claim), 'Download your Response') }}
+  </p>
+{% endmacro %}
+
+{% macro _offerSubmittedForDefendant(claim) %}
+  <h2 class="heading-medium">{{ t('Settle out of court') }}</h2>
+  <p>
+    {{ t('You made an offer to settle the claim out of court. {{ claimantName }} can accept or reject your offer.',
+    {claimantName: claim.claimData.claimant.name}) }}
+  </p>
+
+  <p>{{ t('We’ll email you when they respond.') }}</p>
+
+{% endmacro %}
+
+{% macro _offerAcceptedForDefendant(claim) %}
+  <h2 class="heading-medium">{{ t('Settle out of court') }}</h2>
+  <p>
+    {{ t('The claimant has accepted your offer and signed a legal agreement. '
+    + 'You need to sign the agreement to settle out of court.') }}
+  </p>
+  <p>
+    {{ t('When you’ve both signed the agreement, the claim won’t proceed.') }}
+  </p>
+  <p>
+    <a
+      href="{{ OfferPaths.countersignAgreementPage.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Sign the agreement') }}</a>
+  </p>
+{% endmacro %}
+
+{% macro _offerRejectedForDefendant(claim) %}
+  <h2 class="heading-medium">{{ t('Settle out of court') }}</h2>
+  <p>
+    {{ t('The claimant has rejected your offer to settle the claim. Complete the directions questionnaire.') }}
+  </p>
+
+{% endmacro %}
+
+{% macro _offerSettlementReachedForDefendant(claim) %}
+  <h2 class="heading-medium">{{ t('Agreement signed') }}</h2>
+  <p>
+    {{ t('You’ve both signed a legal agreement. The claim is now settled.') }}
+  </p>
+
+  <p>
+    {{ downloadTheAgreement(claim, DashboardUrlHelper.getStatusUrl(claim)) }}
+  </p>
+{% endmacro %}
+
+{#----------------------------------- CASE HISTORY MACROS-----------------------------------------------#}
```

Comment:
public macros should be at the top private below, usually you are interested in the public ones

Reply:
Great point, moved now 👍